### DISCUSSION TITLE
mjai格式能不能在reviewer里面解析啊

### DISCUSSION THREAD

**Discussion by lewissue** [Equim-chan/mjai-reviewer#80] [General]
有下载mjai格式的功能，我就把一批苏菲谱下载成这个格式了，但是没找到在哪解析>_<难道没有这个功能吗？没有我就重新下载成tenhou.net/6格式。

**Comment by Equim-chan** [comment]
确实还不支持直接读 mjai 牌谱，因为生成 viewer 需要 tenhou.net/6。

**Reply by lewissue** [reply]
感谢解答，那我重新下载一遍
